### DISCUSSION TITLE
Welcome to nxtai Discussions!

### DISCUSSION THREAD

**Discussion by tkrupesh14** [tkrupesh14/nxtai#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RAHULKANTPAL** [comment]
Hello

**Comment by httpsumita** [comment]
Hey I saw your post that I can contribute using Prompt and I have experience with Prompt engineering and Ai integrations. But got a bit confused seeing the whole code written in js, and css. Can you tell me if I need to build it from scratch or if I am missing something?
